packages/ui/src/components/atoms/LangDisplay.tsx

```diff
@@ -1,12 +1,31 @@
 import React from 'react';
 
+import { theme } from '../../themes/theme.styled';
+
 interface LangDisplayProps {
   text: string;
   variables?: object;
 }
 
-const getText = (templateStr: string, variables = {}) =>
-  templateStr.replace(/\${(.*?)}/g, (x, g) => (variables as any)[g] ?? '');
+const getMap = (matchedString: string, g: any, variables: any) => {
+  const resultStr = matchedString.slice(2, matchedString.length - 1);
+
+  if (matchedString.startsWith('$')) return variables[g] ?? '';
+
+  if (matchedString.startsWith('#'))
+    return `<span style="background: ${theme.palette.golden}; -webkit-text-fill-color: transparent; -webkit-background-clip: text;">${resultStr}</span>`;
+  return '';
+};
+
+const getText = (templateStr: string, variables = {}) => {
+  const str = templateStr.replace(/\${(.*?)}|#{(.*?)}/g, (matchedString, g) =>
+    getMap(matchedString, g, variables),
+  );
+
+  return React.createElement('span', {
+    dangerouslySetInnerHTML: { __html: str },
+  });
```

Comment:
Confirm that changing it to `span` does not break anything in the existing screens.